src/game_initialization/multiplayer.cpp

```diff
@@ -682,8 +682,8 @@ void start_local_game_commandline(CVideo& video, const config& game_config, save
 	}
 
 	std::string label = "";
```

Comment:
well yes ofc removing the  `= ""` wouldn't make the code worse but if the variable is unused is should be removed.